content/en/blog/_posts/2022-07-12-gateway-api-in-beta.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: blog
+title: Kubernetes Gateway API Graduates to Beta
+date: 2022-07-12
+slug: gateway-api-first-beta
+canonicalUrl: https://gateway-api.sigs.k8s.io/blog/2022/introducing-v1beta1/
+---
+
+**Author:** Shane Utt (Kong)
+
+Kubernetes SIG Networking is pleased to announce that key Gateway API resources
+are graduating to beta with the `v0.5.0` release of the API. In addition to
+[GatewayClass][gwc], [Gateway][gw] and [HTTPRoute][httpr] graduating to beta,
+this release introduces several new concepts to the API that we'll cover below.
+
+[gwc]:https://gateway-api.sigs.k8s.io/api-types/gatewayclass/
+[gw]:https://gateway-api.sigs.k8s.io/api-types/gateway/
+[httpr]:https://gateway-api.sigs.k8s.io/api-types/httproute/
+
+## What is Gateway API
+
+Gateway API is a collection of resources centered around [Gateway][gw] resources
+(which represent the underlying network gateways / proxy servers) to enable
+robust Kubernetes service networking through expressive, extensible and
+role-oriented interfaces that are implemented by many vendors and have broad
+industry support.
+
+Originally conceived as a successor to the well known [Ingress][ing] API, the
+benefits of Gateway API include (but are not limited to) explicit support for
+many commonly used networking protocols (e.g. `HTTP`, `TLS`, `TCP`, `UDP`) as
+well as tightly integrated support for Transport Layer Security (TLS). The
+`Gateway` resource in particular enables implementations to manage the lifecycle
+of network gateways as a Kubernetes API.
+
+If you're an end-user interested in some of the benefits of Gateway API we
+invite you to jump in and find an implementation that suits you. At the time of
+this release there are over a dozen [implementations][impl] for popular API
```

Comment:
Should we highlight integrations as well? I think we have flagger and cert-manager. It shows the widespread adoption pretty well I think

Reply:
I think it's OK since the linked implementations page also includes integrations, but open to ways to more clearly reference those here as well